scripts/test.sh

```diff
@@ -11,6 +11,9 @@ for f in $( git diff-tree $TRAVIS_COMMIT --name-status -r | grep php | grep -v "
 
 ./scripts/lint.sh
 
+export TERMINUS_CACHE_DIR="$HOME/.terminus/testcache"
+export TERMINUS_TOKENS_DIR="$HOME/.terminus/testcache/tokens"
```

Comment:
I don't like the cache dir being hard-coded. You don't need to define TERMINUS_TOKENS_DIR because it will automatically = TERMINUS_CACHE_DIR/tokens.